add virtual usermeta to /me

HusseinTaha opened the pull request:
I used the usermetaRouter .getBulkForUser method.
```
const keys: string[] = ['country', 'yearofbirth', '$chatdisabled', '$blocked', 'location',
            'singleplayerstats', 'name'];
        options.rootUsermetaClient.getBulkForUser(params, keys, (err2, reply2) => {
```
since the bulk-usermeta is well tested before, I am not sure If i need to add more tests here.
I just adjusted the test and add test the returned metadata as per the defined keys.
Closes #75

j3k0 commented:
- Looks fine. It'd be extra nice if the test actually check that the values are correctly read from the usermeta module. In particular I think about protected values.
- I also realized that `"name"` is already added to the response to `/me` (in the root) so it can be removed from the metadata.

HusseinTaha commented:
Fixed the issue: 
 in normal fake test or test-double
its getting some keys from the central and some other keys from the local .. 
I implemented the integated tests with a fake rest server .. 
so now all the keys are going to one usermeta client => i don't know which one (central, or local) but its only going to one.
.

added integrated tests, with different server for local and central user meta
